tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/AppWrap.scala

```diff
@@ -0,0 +1,85 @@
+package at.forsyte.apalache.tla.pp
+
+import at.forsyte.apalache.tla.lir.oper.{BmcOper, TlaOper}
+import at.forsyte.apalache.tla.lir._
+import at.forsyte.apalache.tla.lir.transformations.{TlaExTransformation, TlaModuleTransformation, TransformationTracker}
+
+/** Replaces instances of user-defined operator applications with a LET-IN wrapper.
+  *
+  * Example:
+  * A(x,y) ~~> LET App_1 == A(x,y) IN App_1
+  *
+  * Operator constants and formal parameters are ignored.
+  */
+class AppWrap(
```

Comment:
Can we give this class a more intuitive name? Like `OperAppToLetInDef`? Essentially, it introduces a symbolic constant for an operator application.